CONTRIBUTING.md

```diff
@@ -1,6 +1,15 @@
 # Contributing
 
-Contribution in the form of [Issues] and [PRs] are welcome.
+Contribution in the form of [PRs] are welcome.
+
+Why We Are No Longer Accepting Public Issues
```

Comment:
nit: Without two spaces at the end of this line, the markdown file will show this in the same line as the text below it. We either want to change this to:
```suggestion
Why We Are No Longer Accepting Public Issues  
```

or 

```suggestion
## Why We Are No Longer Accepting Public Issues
```